src/algebra/lie/basic.lean

```diff
@@ -944,6 +947,13 @@ by { rw [← mem_coe_submodule, sup_coe_to_submodule, submodule.mem_sup], exact
 lemma eq_bot_iff : N = ⊥ ↔ ∀ (m : M), m ∈ N → m = 0 :=
 by { rw eq_bot_iff, exact iff.rfl, }
 
+lemma of_bot_eq_bot (N : lie_submodule R L ↥(⊥ : lie_submodule R L M)) : N = ⊥ :=
```

Comment:
I actually meant to take `N` and `N'` and prove `N = N'`. But I guess you could also mark this `simp` and call it a day.

Reply:
Oh I see; I have now added this.

Also, for reasons I only partly understand, the linter objects to `of_bot_eq_bot` carrying `simp`. If I have time over the weekend I might try to understand this better.